### DISCUSSION TITLE
Machine learning

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/Artificial-Intelligence-Lab#40] [Q&A] [answered]
For which technique output variable will be categorical?

**Comment by TRahulsingh** [comment]
The output variable is categorical in Classification problems, where the goal is to predict labels or classes, such as "yes/no", "male/female", or "product categories".

**Comment by 17-Manish** [comment, marked as answer]
if you are trying to categorise something like determining if an email is spam ore not the output is categorical
